docs/startup/hello-samza/versioned/index.md

```diff
@@ -61,7 +61,7 @@ Then, you can continue w/ the following command in hello-samza project:
 {% highlight bash %}
 mvn clean package
 mkdir -p deploy/samza
-tar -xvf ./target/hello-samza-0.13.1-SNAPSHOT-dist.tar.gz -C deploy/samza
+tar -xvf ./target/hello-samza-0.13.1-dist.tar.gz -C deploy/samza
```

Comment:
This should not be required until the next release. The release process removes the snapshot version with the assumption that you'd be testing against the released samza. The /latest/ copy of this tutorial assumes you're going to use master and build locally, which will create a snapshot version. 

Am I mistaken on this?